kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -1064,17 +1067,13 @@ class ResidualBasedNewtonRaphsonStrategy
 
     /**
      * @brief This method prints information after reach the max number of iterations
-     * @todo Replace by logger
      */
 
     virtual void MaxIterationsExceeded()
     {
-        if (this->GetEchoLevel() != 0 && BaseType::GetModelPart().GetCommunicator().MyPID() == 0)
-        {
-            std::cout << "***************************************************" << std::endl;
-            std::cout << "******* ATTENTION: max iterations exceeded ********" << std::endl;
-            std::cout << "***************************************************" << std::endl;
-        }
+        KRATOS_INFO_IF("NR-Strategy", this->GetEchoLevel() > 0)
```

Comment:
Wouldn't it be nicer to have this printed irrespective of the echo level as a warning?

Reply:
we can, but @loumalouomega was already saying that we have too much output

Btw the python-solvers (at least fluid & structure) do already print a warning like this (independednt of echo-lvl):
~~~
::[MechanicalSolver]::  Solver did not converge for step 1
~~~